app/views/audit/audit.html

```diff
@@ -1,7 +1,11 @@
 {% extends "includes/layout.html" %}
+
 {% block page_title %}
```

Comment:
I suspect these may all need changing to `pageTitle` to use the new layout. Possibly easier to do what Tom suggested, duplicate the `includes/layout.html` as `includes/layout_new.html` and make new changes to that?

Then start using the new template on new pages, once you've move over you can remove the old template, rename `layout_new.html` then just search/replace for a single PR.